userprefs.php

```diff
@@ -446,7 +446,6 @@ function save_general_tab() {
 
     $userSettings->set_boolean('hide_special_colors', $_POST["show_special_colors"]=='no');
 
-    echo mysqli_error(DPDatabase::get_connection());
```

Comment:
Why delete? We don't want an unconditional
   `echo DPDatabase::log_error();`
of course, but we could say
   `mysqli_query(...) or print(DPDatabase::log_error())`
to reproduce the current behavior.

Reply:
This one in particular is only going to print and return the error from the last DB call in the `$userSettings->set_bool()` which is arguably not what the user actually intended -- probably because I or someone else didn't clean this up after a refactor. I'll re-attach this one to the query call and fix the others.

Reply:
I checked the history: the `mysql_query` got separated from the `mysql_error` way back on 2005-03-19, when the userSettings code was inserted.